tensorboard/webapp/widgets/data_table/content_row_component.ts

```diff
@@ -0,0 +1,35 @@
+/* Copyright 2022 The TensorFlow Authors. All Rights Reserved.
```

Comment:
/* Copyright 2023 The TensorFlow Authors. All Rights Reserved.

Reply:
Done.